index.html

```diff
@@ -0,0 +1,99 @@
+                    <a href="https://www.facebook.com/CryptoCon2020/" target="_blank" rel="noopener noreferrer">
+                        <img src="img/facebook.png" alt="Facebook"></a>
+                    <a href="#"><p>English</p></a>
+                    <a href="#"><p>My Page</p></a>
+                    <a href="#"><p>Logout</p></a>
+                </div>
+            </div>
+            <nav>
+                <img id="toggle-nav" src="img/burger-bar.png" alt="Toggle Menu">
+                <div class="nav__menu" id="nav-menu">
+                    <ul class="nav__links">
+                        <li><a href="about.html" class="nav__link">About</a></li>
+                        <li><a href="#prgm" class="nav__link">Program</a></li>
+                        <li><a href="#spks" class="nav__link">Speakers</a></li>
+                        <li class="box"><a href="#spns" class="nav__link">Sponsors</a></li>
+                    </ul>
+                    <img id="close-nav" src="img/close.png" alt="Close Navigation">
+                </div>
+            </nav>
+        </div>
+    </header>
+    <section class="hero__container" id="home">
+        <p class="hero__hello">"Hello! Sharing world"</p>
+        <div class="hero__headline">
+            <h1>Crypto-con <br> global blockchain summit <br> 2022</h1>
+        </div>
+        <p class="hero__welcome">Welcome to our blockchain conference we focus on solving problems while leveraging the
+            blockchain
+            technology.
+            Our speakers are people whose goal is change in how business and society works today.</p>
+        <h3 class="hero__date">2022.04.13(THUR) ~ 16(FRI)</h3>
+        <p class="hero__location">@ Kenya International Conference Center, Nairobi Kenya</p>
+    </section>
+</div>
+
+<section id="prgm" class="program">
+    <div class="program__container">
+        <h2>Main Program</h2>
+        <hr>
+        <div id="program" class="program__content">
+        </div>
+        <button type="button">Book your ticket now</button>
+    </div>
+</section>
+
+<section id="spks" class="speakers">
+    <div class="speakers__container">
+        <h2>Featured Speakers</h2>
+        <hr>
+        <div id="speakers" class="speakers__content">
+        </div>
+    </div>
+</section>
+
+<div id="spns" class="sponsors">
+    <div class="sponsors__container">
+        <h2>Partners</h2>
+        <hr>
+        <div id="sponsors" class="sponsors__content">
+            <img src="img/tron.svg" alt="Tron sponsor">
+            <img src="img/solana-logo.svg" alt="Solana Sponsor">
+            <img src="img/peercoin.svg" alt="Peer coin sponsor">
+            <img src="img/cardano.svg" alt="Tron sponsor">
+            <img src="img/solana-logo.svg" alt="Solana Sponsor">
+        </div>
+    </div>
+</div>
+<footer>
+    <img src="img/icons8-blockchain.png" alt="Logo">
+    <p>Copyright &copy; 2020 Crypto Con</p>
+</footer>
```

Comment:
- [ ] **Required:** You should space the footer according to the Behance template.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/162812203-b5a78d00-3f08-44a0-bb0a-5bbc99fa9ecf.png) | ![image](https://user-images.githubusercontent.com/27709832/162812324-8fb04acf-267a-4931-b02d-420353d6c3d9.png)